internal/oci/runtime_oci.go

```diff
@@ -965,8 +965,9 @@ killContainer:
 		if _, err := r.runtimeCmd("kill", c.ID(), "KILL"); err != nil {
 			if !errors.Is(err, ErrNotFound) {
 				log.Errorf(ctx, "Killing container %v failed: %v", c.ID(), err)
+			} else {
+				log.Debugf(ctx, "Error while killing container %s: %v", c.ID(), err)
```

Comment:
@haircommander, wasn't this part of some changes that @sohankunkerkar added some time ago?

My recollection might be wrong here, but this does look familiar, a bit.

Reply:
this is similar to https://github.com/cri-o/cri-o/pull/8431 though it's an additional case

Reply:
Ah, I see. You prefer to have if-else there, no worries. We actually left it to be a catch-all on purpose in the past.